br.com.wiselabs.simplexity.desktop

```diff
@@ -0,0 +1,12 @@
+[Desktop Entry]
+Version=1.2
+Type=Application
+Name=Simplexity
+GenericName=Perplexity.AI app
+Comment=Simple and safe Perplexity.AI desktop client
+Exec=simplexity
```

Comment:
```suggestion
Exec=run.sh
```

Reply:
File moved to upstream and fixed to call run.sh instead binary directly.